setup.py

```diff
@@ -21,6 +21,9 @@
     'intervaltree == 3.0.2',
     'sox == 1.3.7',
     'PGet == 0.5.0',
+    'tqdm',
+    'chardet',
```

Comment:
Always fix the version.

```suggestion
    'chardet == 3.0.4',
```